dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaHostComponentsTask.java

```diff
@@ -57,10 +57,32 @@ protected void doRun(
 
         try (CloseableHttpResponse response = httpClient.execute(new HttpGet(hostsComponentsUrl))) {
           JsonNode json = getObjectMapper().readTree(response.getEntity().getContent());
-          writer.write(json.toString());
+          writer.write(wrapResponseWithHostId(host.getId(), json));
           writer.write('\n');
         }
       }
     }
   }
+
+  private String wrapResponseWithHostId(String hostId, JsonNode payload) throws Exception {
+    return getObjectMapper().writeValueAsString(new PayloadEnrichedWithHostId(hostId, payload));
+  }
+
+  private static class PayloadEnrichedWithHostId {
+    private final String hostId;
+    private final JsonNode payload;
+
+    public PayloadEnrichedWithHostId(String hostId, JsonNode payload) {
+      this.hostId = hostId;
+      this.payload = payload;
+    }
+
+    public String getHostId() {
```

Comment:
Will you use these getters in future?

Reply:
I don't, but `ObjectMapper` has such plans.

Reply:
Change to use `@JsonProperty` to avoid possible confusions.